src/network/NetworkTypes.h

```diff
@@ -18,6 +18,7 @@
 
 #include <SDL_endian.h>
 #include <SDL_platform.h>
+#include "../common.h"
```

Comment:
You can't do this, for the comment below about winsock2 being included before windows.h.